docs/overview.adoc

```diff
@@ -155,14 +155,26 @@ Returns a virtual graph that represents the labels and relationship-types availa
 | CALL apoc.meta.stats  yield labelCount, relTypeCount, propertyKeyCount, nodeCount, relCount, labels, relTypes, stats | returns the information stored in the transactional database statistics
 |===
 
-.Functions
+.Functions Deprecated
 [cols="1m,5"]
 |===
-| apoc.meta.type(value) | type name of a value (`INTEGER,FLOAT,STRING,BOOLEAN,RELATIONSHIP,NODE,PATH,NULL,UNKNOWN,MAP,LIST`)
+| apoc.meta.type(value) | type name of a value (`INTEGER,FLOAT,STRING,BOOLEAN,RELATIONSHIP,NODE,PATH,NULL,MAP,LIST`)
 | apoc.meta.isType(value,type) | returns a row if type name matches none if not
 | apoc.meta.types(node or relationship or map) | returns a a map of property-keys to their names
 |===
 
+If no type was found, the function return name of the class.
+
+.Functions
+[cols="1m,5"]
+|===
+| apoc.meta.v2.type(value) | type name of a value (`INTEGER,FLOAT,STRING,BOOLEAN,RELATIONSHIP,NODE,PATH,NULL,MAP,LIST,POINT,DATE,DATE_TIME,LOCAL_TIME,LOCAL_DATE_TIME,TIME,DURATION`)
```

Comment:
I think we need this new function and the deprecation because the outputs are different, for instance:
From `apoc.meta.types`
```
"LOCAL_TIME" -> "LocalTime"
"DATE" -> "LocalDate"
"NULL" -> "NULL"
"LOCAL_DATE" -> "LocalDateTime"
"TIME" -> "OffsetTime"
"DATE_TIME" -> "ZonedDateTime"
```
From `apoc.meta.v2.types`
```
"LOCAL_TIME" -> "LOCAL_TIME?"
"DATE" -> "DATE?"
"NULL" -> "NULL?"
"LOCAL_DATE" -> "LOCAL_DATE_TIME?"
"TIME" -> "TIME?"
"DATE_TIME" -> "DATE_TIME?"
```